integration-tests/chopsticks/src/network/hydroDx.ts

```diff
@@ -0,0 +1,54 @@
+import { setupContext, SetupOption } from '@acala-network/chopsticks-testing'
+import type { Config } from './types.js'
+import { u8aToHex } from '@polkadot/util'
+import { decodeAddress } from '@polkadot/util-crypto'
+import * as SpiritnetConfig from './spiritnet.js'
+
+export const options: SetupOption = {
+	endpoint: ['wss://hydradx-rpc.dwellir.com', 'wss://rpc.hydradx.cloud'],
```

Comment:
And here.

Reply:
[46f5119](https://github.com/KILTprotocol/kilt-node/pull/614/commits/46f511924ffda667b943584517f5d87b9a5f3bcb)